src/index.native.animated.js

```diff
@@ -135,8 +151,19 @@ class SwipeableViews extends Component {
     }
   }
 
+  setIndexWithoutTransition(newIndex) {
```

Comment:
That sounds like an imperative method. Instead, what about using a property to disable the transition?